docs/contribution.md

```diff
@@ -0,0 +1,148 @@
+coordinator generates and publishes a challenge file, which the first participant downloads, generates some entropy 
+(the so-called toxic waste). Then, the participant runs a computation to produce a new challenge file, which is made 
+available for coordinator verification. If the contribution has been verified as correct, the next participant will 
+repeat the process starting from this latest contribution, otherwise is going to recompute the work taking the 
+position of the previous participant. The process is repeated indefinitely until the coordinator decides to end 
+the ceremony or a predetermined end date is reached. Ultimately, the ceremony results in the extraction of a key 
+for the specific circuit being used.
+
+
+## Requirements
+
+1. **A computer**: 
+
+A computer with 8GB or above Space and Ram is recommended. Ideally Mac or Linux, Windows via 
+[WSL](https://learn.microsoft.com/en-us/windows/wsl/install) is Ok;
+
+2. **A strong internet connection**:
+
+The most common cause of failures is an inability to upload your contribution to the ceremony before the timeout 
+period expires. Please run the ceremony with a strong internet connection, and your upload and download bandwidths 
+should be at least 25 Mbps each (preferably 50+ Mbps each). Using a wired (ethernet) connection rather than a 
+wireless (WiFi) connection is also recommended.
+
+3. **A GitHub Accout**:
+
+A GitHub account must be a month old with at least one follower, five followings and two repos.
+
+4. **Node Version Manager (NVM)**
+
+Running the CLI app to contribute to the ceremony requires node version 16.20 or above.
+We’ll install nvm to change node versions by downloading a shell script using the following command in the terminal:
+```bash
+curl -sL https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.0/install.sh -o install_nvm.sh
+sh install_nvm.sh
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+```
+
+5. **Install node v18**
+   
+Before contribution, let’s install and switch to node v18 which the ceremony relies on.
+```bash
+nvm install 18
+nvm use 18
+```
+
+## Contributing to the Ceremony
+
+1. **Create a temporary directory**
+```bash
+mkdir ~/phase2cli
+cd ~/phase2cli
+```
+
+2. **Install phase2cli**
+
+```bash
+npm install -g @ethstorage-io/phase2cli
+```
+
+3. **Authenticate with GitHub**
+
+If your GitHub meets the requirements, you will be eligible for this next step using the following command
+```bash
+phase2cli auth
+```
+You will be provided [a link](https://github.com/login/device) and your authentication code as following.
+
+<img alt="auth0.png" src="auth0.png" width="100%"/>
+
+Open the link in your browser and log in your GitHub with your authentication code.
+
+<img alt="auth1.png" src="auth1.png" width=40%/>
+
+4. **Contribute**
+
+```bash
+phase2cli contribute
+```
+
+Run the command and select the circom you want to contribute (like _EthStorage circom_);
```

Comment:
Are there multiple choices in a product environment as in the screenshot? Need explanations for each if yes.

Reply:
Circoms already have explanations when users are selecting.